src/components/common/AddressInput/useNameResolver.ts

```diff
@@ -1,7 +1,7 @@
 import { useMemo } from 'react'
 import { useWeb3ReadOnly } from '@/hooks/wallets/web3'
 import useAsync from '@/hooks/useAsync'
-import { isDomain, resolveName } from '@/services/domains'
```

Comment:
Unrelated but I also renamed domains.ts to ens.ts because `domain` was confusing. I thought it was related to domain parsing or something.